examples/tick-ami/tick.json

```diff
@@ -123,6 +123,15 @@
   },{
     "type": "shell",
     "inline": [
+       "sudo apt-get install -y jq dos2unix",
+      "dos2unix /tmp/terraform-aws-influx/modules/install-telegraf/install-telegraf",
+      "dos2unix /tmp/terraform-aws-influx/modules/install-influxdb/install-influxdb",
+      "dos2unix /tmp/terraform-aws-influx/modules/install-chronograf/install-chronograf",
+      "dos2unix /tmp/terraform-aws-influx/modules/install-kapacitor/install-kapacitor",
```

Comment:
What do the line endings have to do with the permissions issue you saw?

Reply:
This is a separate issue, but also caused by Windows. Once the scripts are cloned to a Windows machine, their line endings are changed by the operating system, and those line endings aren't recognized by Unix operating systems, so the bash scripts will throw a lot of errors about line endings and won't run.